### ISSUE TITLE
GitHub star button has incorrect link location for repos outside canjs org

### ISSUE THREAD

**Issue by marshallswain** [bit-docs/bit-docs-tag-package#1]
The `@package` tag that should be pointing here: https://github.com/feathersjs/authentication-popups is instead pointing to canjs/authentication-popups.

Used in this PR: https://github.com/canjs/canjs/pull/2663

**Comment by leoj3n** [comment]
I'll look at this tomorrow.

**Comment by imaustink** [comment]
@justinbmeyer, @marshallswain

I've reviewed what's involved in fixing this and there are a few different options. The main issue is that the Github organization isn't readily available in the package.json file. The closest thing would be the repository field.

I talked with @leoj3n and between us, we considered the following possible solutions.

1. Modify the `@package` tag to parse the repository URL for its organization/username and attach that to the object it returns.
2. Add a new `@org` tag that would allow you to define and Github organization.
3. Register a helper within `bit-docs-generate-html`  to allow something like this `{{getOrg package.repository}}` to work.
4. Same as number 3, except the helper, would be registered to `bit-docs-html-canjs` to isolate functionality to that custom template.
